sdk/storage/storage-blob/test/README.md

```diff
@@ -0,0 +1,22 @@
+# Testing
+
+To test this project, make sure to first build it properly by following our [building instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#building). Once the project is correctly built, you will be able to run the tests by following the [testing instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#testing).
+
+You can use existing Azure resources for the live tests, or generate new ones by using our [New-TestResources.ps1](https://github.com/Azure/azure-sdk-for-js/blob/master/eng/common/TestResources/New-TestResources.ps1) script, which will use an [ARM template](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/test-resources.json) that already has all of the the necessary configurations.
+
+The Azure resources that are used by the tests in this project are:
+
+- Two general purpose, v2 [Azure Storage](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview) account properly configured to test the Azure Storage Blob, Azure Storage Queue, Azure File Data Lake and Azure File Share services.
+
+You will also need to set the below environment variables to run the live tests:
+
+- `TEST_MODE`: Should have `live` assigned.
+- `ACCOUNT_NAME`: The Azure Storage account name.
+- `STORAGE_DATALAKE_ACCOUNT_NAME`: An Azure Storage account name that has Azure Storage File Data Lake features.
+- `ACCOUNT_KEY`: The Azure Storage account key.
+- `ACCOUNT_SAS`: The shared access signature.
+- `STORAGE_CONNECTION_STRING`: The Azure Storage account connection string.
```

Comment:
@jeremymeng, ACCOUNT_TOKEN ? And anything else that is missed in here?

Reply:
I took these environment variables form: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/test-resources.json

Perhaps that file needs to be updated?

Reply:
If the ACCOUNT_TOKEN is missed, some of the tests will be skipped and I believe that's what is happening for the nightly test runs.

Reply:
Seems like we should fix that!

Reply:
I remember it's for manual testing. We would need to request an access token using Azure CLI or similar tool. It's okay to not list it here. /cc @XiaoningLiu @jiacfan @ljian3377